flow/designs/nangate45_fakelvt/jpeg/metadata-base-ok.json

```diff
@@ -0,0 +1 @@
+../../nangate45/jpeg/metadata-base-ok.json
```

Comment:
Wouldn't you expect this to differ?  Same with rules-base.json

Reply:
Since I was only looking at timing, it did not seem necessary. We can always change it later if needed for a more robust test.

Reply:
Fwiw the metrics contain slacks

Reply:
Learned something new. I will update this file after running the baseline flow.